src/components/desktop/Header.js

```diff
@@ -48,28 +94,106 @@ const StyledHeader = styled.header`
           font-family: inherit;
           font-weight: normal;
           cursor: pointer;
+          background-color: ${({ theme }) => theme.background};
         }
 
         button,
         &.active {
-          background-color: ${({ theme }) => theme.background};
           color: ${({ theme }) => theme.orange};
         }
       }
     }
   }
 `
+const Dropdown = ({ className, children, text }) => (
+  <div className={className}>
+    <div className={OnAbout() ? 'active' : 'button'}>
```

Comment:
The `.button` class should always apply here (i.e. this is the same element whether the dropdown is active or not)

Reply:
See above about calling a nested function with a hook.